admin/client/src/components/CompositeField/README.md

```diff
@@ -0,0 +1,19 @@
+# CompositeField
+
+For containing groups of fields in a container element.
+
+## Example
+
+```
+<CompositeField id="Container">
+    <TextField name="FirstName" />
+    <TextField name="LastName" />
+</Tabs>
+```
+
+## Properties
+
+ * `name` (string) (required): A key to match with the Tabset `activeKey` property to show the content.
+ * `title` (string): The label to display for the Tab, can be set `null` to hide the Tab.
```

Comment:
"tab" references don't make sense out of context here (you only mention "composite field" in the intro)